### DISCUSSION TITLE
Vitest example for individual components testing

### DISCUSSION THREAD

**Discussion by thomasverleye** [remix-run/remix#3155] [Q&A] [answered]
Hello 👋 

We've been trying to set up a simple dummy component test with the Indie Stack setup but it looks like something is misconfigured in Typescript.

```tsx
// /app/component/Dummy/index.tsx
function Dummy() {
  return <h1>{`Hello World`}</h1>;
}

export default Dummy;
```

```tsx
// /app/component/Dummy/index.test.ts
import { render } from '@testing-library/react';
import Dummy from './index';

test('Try to render the Dummy Component', () => {
  const { container } = render(<Dummy />);
  expect(container).toBeInTheDocument();
});
```

Here are some error messages (got 9 but seems a lot to paste here just to state that TS got things wrong.)
```
'Dummy' refers to a value, but is being used as a type here. Did you mean 'typeof Dummy'?
```

Does anyone here know what has been misconfigured?

It may help to add an example test for components in these stacks 🙇

**Comment by Arsikod** [comment, marked as answer]
change Dummy.ts to Dummy.tsx

**Reply by thomasverleye** [reply]
I had fixed it that way indeed, thanks 🙌
